index.html

```diff
@@ -357,16 +354,19 @@
       <p>The <a>PerformanceObserver</a> interface can be used to observe the
       <a>Performance Timeline</a> and be notified of new performance entries as
       they are recorded by the user agent.</p>
+      <p>Each <a>PerformanceObserver</a> has these associated concepts:</p>
       <ul>
-        <li>A <dfn>registered performance observer</dfn> consists of an
-        observer (a <a>PerformanceObserver</a> object) and options (a
-        <a>PerformanceObserverInit</a> dictionary).
-        </li>
-        <li>Each <a>registered performance observer</a> object has a private
-        list of <a>PerformanceEntry</a> objects (<dfn>observer buffer</dfn>)
-        which is initially empty.
+        <li>A <dfn>callback</dfn> set on creation.</li>
+        <li>A list of <a>PerformanceEntry</a> objects called the <dfn>observer
+        buffer</dfn> that is initially empty.
         </li>
       </ul>
+      <p>The `PerformanceObserver(callback)` constructor must create a new
+      <a>PerformanceObserver</a> object with <a>callback</a> set to
+      <var>callback</var> and then return it.</p>
```

Comment:
Shouldn't this also add it its global environment? Or at least a global environment of sorts.